input/pagecontent/reference.md

```diff
@@ -734,198 +878,251 @@ For convenience and compactness, cardinality rules can be combined with [flag ru
+| TU | TU | Trial use element |
+| D | D | Draft element |
+{: .grid }
 
-```
-* -> "{map}" "{comment}" {mime-type}
-```
+FHIR also defines I and NE flags, representing elements affected by constraints, and elements that cannot have extensions, respectively. These flags are not directly supported in flag syntax, since the I flag is determined by the presence of [invariants](#obeys-rules), and NE flags apply only to infrastructural elements in base resources. If needed, these flags can be set using [caret syntax](#structuredefinition-escape-paths).
+
+The following syntaxes can be used to assigning flags:
 
 ```
-* {path} -> "{map}" "{comment}" {mime-type}
-```
+* <element> {flag}
 
-The first type of rule applies to mapping the profile as a whole to the target specification. The second type of rule maps a specific element to the target.
+* <element> {flag1} {flag2} ...
 
-In these rules, the `"{comment}"` string and `{mime-type}` code are optional. The mime type code must conform to https://tools.ietf.org/html/bcp13 (FHIR value set https://www.hl7.org/fhir/valueset-mimetypes.html).
+* <element1> and <element2> and <element3> ... {flag}
+
+* <element1> and <element2> and <element3> ... {flag1} {flag2} ...
+```
 
 **Examples:**
 
-* Map the entire profile to a Patient item in another specification:
+* Declare communication to be MustSupport and Modifier:
 
   ```
-  * -> "Patient" "This profile maps to Patient in Argonaut"
+  * communication MS ?!
   ```
 
-* Map the identifier.value element from one IG to another:
+* Declare a list of elements and nested elements to be MustSupport:
 
   ```
-  * identifier.value -> "Patient.identifier.value"
+  * identifier and identifier.system and identifier.value and name and name.family MS
   ```
 
->**Note:** Unlike setting the mapping directly in the SD, mapping rules within a Mapping item do not include the name of the resource in the path on the left hand side.
-
-#### Rule Set Rules
+#### Insert Rules
 
 [Rule sets](#defining-rule-sets) are reusable groups of rules that are defined independently of other items. To use a rule set, an `insert` rule is used:
 
 ```
-* insert {rule set name}
+* insert {RuleSet name}
 ```
 
-The rules in the named rule set are evaluated as if they were copied and pasted in the designated location. Rule sets can contain other rule sets, but circular dependencies are not allowed.
+The rules in the named rule set are evaluated as if they were copied and pasted in the designated location.
 
 Each rule in the rule set should be compatible with item where the rule set is inserted, in the sense that all the rules defined in the rule set apply to elements actually present in the target. Implementations should check the legality of a rule set at compile time. If a particular rule from a rule set does not match an element in the target, that rule will not be applied, and an error should be emitted. It is up to implementations if other valid rules from the rule set are applied.
```

Comment:
"with item where the rule set is inserted" -> "with the item where the rule set is inserted"

Reply:
Fixed.